workspace.go

```diff
@@ -0,0 +1,66 @@
+package twilio
+
+import (
+	"context"
+	"net/url"
+)
+
+const WorkspacePath = "Workspaces/"
```

Comment:
I don't think you need this /? I don't see this being needed anywhere else. Is this a quirk on this particular API call? I see it being used only in `DeleteResource`

Reply:
Only the create one needed it with a slash, I'll try removing it and seeing, but there is a lot of hard coded oneoffs in this code for paths